lib/mocha.js

```diff
@@ -483,21 +512,32 @@ Mocha.prototype.run = function(fn) {
   if (options.globals) {
     runner.globals(options.globals);
   }
+  // Use only the first reporter for growl, since we don't want to
+  // send several notifications for the same test suite
   if (options.growl) {
-    this._growl(runner, reporter);
+    this._growl(runner, reporters[0]);
```

Comment:
I have no idea about growl.  we should look at gh bigdata and see if anyone is using it.